### ISSUE TITLE
Email is not sent on recovering a cart to admin

### ISSUE THREAD

**Issue by galakhyati** [TycheSoftwares/Woocommerce-abandon-cart-lite#110]
When a cart is recovered, the admin is not getting an email for recovery. The setting is enabled for sending the email on recovering the cart. This is the case when testing with PHP7

**Comment by Chetna1510** [comment]
I am not able to replicate the same issue on my local with PHP 7. 

I have abandoned a cart as a logged-in user and I have placed an order from the abandoned cart reminder email. Then I (Admin) am able to receive the recovery email for my cart.

See the recovery email: http://screencast.com/t/vLiqx2cbvwol

See the screenshot of order details page: http://screencast.com/t/GJOHjmO97

**Comment by Chetna1510** [comment]
I have tried to replicate the issue with the latest version of github. I am not able to replicate the same issue on my local with PHP 7.1.7.

I have abandoned a cart as a logged-in user and guest user and I have placed an order from the abandoned cart reminder email. Then recovery email is being sent to Admin for my cart.

See the recovery email: https://www.screencast.com/t/fBKT7zZVv
See the screenshot of order details page: https://www.screencast.com/t/J1tLCyTfyoj5
